src/commands.ts

```diff
@@ -1431,7 +1431,23 @@ ${contents}
 				handler.applySuggestion(comment);
 			}
 		}));
-
+	context.subscriptions.push(
+		vscode.commands.registerCommand('pr.applySuggestionWithCopilot', async (comment: GHPRComment) => {
+			/* __GDPR__
+				"pr.applySuggestionWithCopilot" : {}
+			*/
+			telemetry.sendTelemetryEvent('pr.applySuggestionWithCopilot');
+
+			const commentThread = comment.parent;
+			commentThread.hide();
```

Comment:
In insiders, this should work. I'll be removing `hide` in the future. 

```suggestion
			commentThread.collapsibleState = vscode.CommentThreadCollapsibleState.Collapsed;
```

If you do change to use this, then please don't forget to also bump the engine in package.json

Reply:
PR for this: https://github.com/microsoft/vscode-pull-request-github/pull/6275